HW03-VaughtK/hw03_VaughtK.py

```diff
@@ -0,0 +1,182 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Kai Vaught
+# Username: VaughtK
+#
+# Assignment: Homework 3
+# Purpose: To learn how branches work.
+# Google Doc Link: https://docs.google.com/document/d/1OvsmR456y61n3UzwBlIlVE13IBz0tF39Vyv10F24ZGI/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+def house_turtle():
+    import turtle
```

Comment:
It looks like you used a second import of turtle to solve a scoping problem. The correct solution would be to create the turtle in `main()`, and pass it into the function as an input parameter:

```
def draw_house(t):
    t.setpos(0, -100)
    ...
    
    
def main():
    ...
    t = turtle.Turtle()
    draw_house(t)
    ...
```